go/matrix/matrix.go

```diff
@@ -16,48 +16,57 @@ type matrix struct {
 	values [][]int
 }
 
+//  rowParsing parses a matrix row represented as a string, and converts
+//  it to a slice of integer values.
+//  parameters :
+//    * strRow : the string representation of the line, with cells
+//               separated by COLUMN_SEPARATOR
+func rowParsing(strRow string) ([]int, error) {
+	returnedRow := make([]int, 0)
+
+	for _, value := range strings.Split(strRow, COLUMN_SEPARATOR) {
+		intValue, convErr := strconv.Atoi(value)
+		if convErr != nil {
+			return make([]int, 0), convErr
```

Comment:
nit: You could return nil for a slice and it's the preferred method when there is an error :slightly_smiling_face:

Reply:
Good to know, thanks !